src/app/data-client.ts

```diff
@@ -0,0 +1,58 @@
+import { type RpcOptions } from '@improbable-eng/grpc-web/dist/typings/client.d';
+import {
+  DataRequest,
+  Filter,
+  TabularData,
+  TabularDataByFilterRequest,
+  TabularDataByFilterResponse,
+} from '../gen/app/data/v1/data_pb';
+import { DataServiceClient } from '../gen/app/data/v1/data_pb_service';
+import { promisify } from '../utils';
+
+export class DataClient {
+  private client: DataServiceClient;
+
+  constructor(serviceHost: string, grpcOptions: RpcOptions) {
+    this.client = new DataServiceClient(serviceHost, grpcOptions);
+  }
+
+  private get service() {
+    return this.client;
+  }
```

Comment:
Is the `service` getter required if the `client` private instance property is renamed to `service`?

```suggestion
  private service: DataServiceClient;

  constructor(serviceHost: string, grpcOptions: RpcOptions) {
    this.service = new DataServiceClient(serviceHost, grpcOptions);
  }
```

Since it's a private value, I'm curious about the need for a private getter that just returns the DataServiceClient instance.

Reply:
i actually don't remember why we started using this pattern... we don't need this extra function